automation/submission-perf-inc.sh

```diff
@@ -240,30 +241,15 @@ monitorSubmission() {
 
     printf "\nFetching status for submission ID '%s':" "${submissionId}"
 
-    submissionStatus=$(
-        curl \
+    submissionDetails=$(curl \
             -X GET \
             --header 'Accept: application/json' \
             --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-            "https://firecloud-orchestration.dsde-${ENV}.broadinstitute.org/api/workspaces/${namespace}/${name}/submissions/${submissionId}" \
-        | jq -r '.status'
-    )
-    workflowsStatus=$(
-        curl \
-            -X GET \
-            --header 'Accept: application/json' \
-            --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-            "https://firecloud-orchestration.dsde-${ENV}.broadinstitute.org/api/workspaces/${namespace}/${name}/submissions/${submissionId}" \
-         | jq -r '.workflows[] | .status'
-    )
-    workflowFailures=$(
-        curl \
-            -X GET \
-            --header 'Accept: application/json' \
-            --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-            "https://firecloud-orchestration.dsde-${ENV}.broadinstitute.org/api/workspaces/${namespace}/${name}/submissions/${submissionId}" \
-         | jq -r '[.workflows[] | select(.status == "Failed")] | length'
-    )
+            "https://firecloud-orchestration.dsde-${ENV}.broadinstitute.org/api/workspaces/${namespace}/${name}/submissions/${submissionId}")
+
+    submissionStatus=$(jq -r '.status' <<< "${submissionDetails}")
+    workflowsStatus=$(jq -r '.workflows[] | .status' <<< "${submissionDetails}")
+    workflowFailures=$(jq -r '[.workflows[] | select(.status == "Failed")] | length' <<< "${submissionDetails}")
```

Comment:
Just a heads up, when I was working on fixing some other issue, this approach where the details are fetched once and then interpreted locally was causing the tests to fail with below error. Not sure why it was getting `workflows[]` as empty. Hence I had reverted it back to fetching details 3 times. Not sure if this is still happening. Builds where I saw the below error - [#52](https://fc-jenkins.dsp-techops.broadinstitute.org/job/alpha-impossible-to-read-archived-metadata-from-within-workflow-check/52/console) and [#1199](https://fc-jenkins.dsp-techops.broadinstitute.org/job/PerformanceTest-against-Alpha/1199/console)
 ```
jq: error (at <stdin>:8): Cannot iterate over null (null)
```

Reply:
Thanks for the heads up. I removed this change so it's just about logging now